fiboa_cli/datasets/fs_za_ct.py

```diff
@@ -0,0 +1,144 @@
+# TEMPLATE FOR A FIBOA CONVERTER
+#
+# Copy this file and rename it to something sensible.
+# The name of the file will be the name of the converter in the cli.
+# If you name it 'de_abc' you'll be able to run `fiboa convert de_abc` in the cli.
+
+from ..convert_utils import convert as convert_
+
+# File to read the data from
+# Can read any tabular data format that GeoPandas can read through read_file()
+# Supported protcols: HTTP(S), GCS, S3, or the local file system
+
+# Local URI added to the repository for initial conversion, Original Source https://beta.source.coop/esa/fusion-competition/
+URI = "/home/byteboogie/fieldscapes/south_africa/fs_za_ct.gpkg"
+
+# Unique identifier for the collection
+ID = "fs_za_ct"
+# Title of the collection
+TITLE = "Field boundaries for Cape Town, South Africa"
+# Description of the collection. Can be multiline and include CommonMark.
+DESCRIPTION = """ The dataset contains field boundaries for the Cape Town, South Africa."""
+# Bounding box of the data in WGS84 coordinates
+BBOX = [20.521492384730347, -34.39922362572791, 21.04341451023305, -33.980506187460875]
+
+# Provider name, can be None if not applicable, must be provided if PROVIDER_URL is provided
+PROVIDER_NAME = "Planet, Radiant Earth Foundation, Western Cape Department of Agriculture, & German Aerospace Center (DLR)"
+# URL to the homepage of the data or the provider, can be None if not applicable
+PROVIDER_URL = "https://beta.source.coop/esa/fusion-competition/"
+# Attribution, can be None if not applicable
+ATTRIBUTION = "ESA Fusion Competition"
+
+# License of the data, either
+# 1. a SPDX license identifier (including "dl-de/by-2-0" / "dl-de/zero-2-0"), or
+LICENSE = "CC BY-NC-SA 4.0"
```

Comment:
According to https://spdx.github.io/spdx-spec/v2.3/SPDX-license-list/ this should be:
```suggestion
LICENSE = "CC-BY-NC-SA-4.0"
```

Reply:
Yes I forgot to follow the SPDX format